src/main/java/Duke.java

```diff
@@ -36,7 +36,8 @@ public static void list() {
         System.out.println(line);
```

Comment:
maybe "line" can be replaced with "dottedLine"? I was a bit confused as to what this was referring to.